documentation/git_and_github_setup.md

```diff
@@ -0,0 +1,67 @@
+# **Git Training: Pre-Training Git and GitHub Access Checklist for Mobility Metrics Data Team**
+
+This guide will walk through the steps to ensure that you have access to Git and Github. For those that are active users this will not apply to you.
+
+## Setting up a GitHub account and installing Git software 
+
+### Returning Contributers 
+
+Please make sure you still have access to your GitHub account. You can check this navigating to https://github.com/ and checking if you still can view the [mobility-from-poverty](https://github.com/UI-Research/mobility-from-poverty) project repository page. If you cannot access the repository, please email me at jwalsh@urban.org describing the issue. 
+
+Please check if Git is downloaded on your computer. Reminder, if Git is downloaded you should see an option to “Open Git Bash here” when you left click in any folder on your computer. If Git is not installed on your current computer, please install it from the following website.  
+
+If you still have the mobility-from-poverty repository cloned to your local computer, please delete that folder. We will be re-cloning the latest version of the repository from GItHub during this training.  
+
+ 
+### New Contributors
+
+If you have not so already, please create a [GitHub](https://github.com/) account.  
+
+After creating the github account ask, them to fill out the Urban GitHub [intake form](https://app.smartsheet.com/b/form/9f0c5ba330dd4b73980fe5a6e17216b5) requesting access to UI Research. After recieving access you should be able to access the [mobility-from-poverty](https://github.com/UI-Research/mobility-from-poverty) project repository page. 
+
+If you have not already installed the Git software, please complete the installation process from the following [website](https://git-scm.com/downloads). You will know it installed successfully if upon completion when you left click in any folder on your computer you see an option to “Open Git Bash here”. 
```

Comment:
This will probably require the Helpdesk. Though it should be installed on all work computers.